apps/settings/js/panels/feedback_send/feedback_send.js

```diff
@@ -136,9 +141,6 @@ define(function(require) {
     },
 
     _responseHandler: function() {
-      if (this._xhr.readyState !== 4) {
-        return;
-      }
       switch (this._xhr.status) {
```

Comment:
The cases in this switch statement are one of the issues I mentioned in the bug comments. An HTTP 400 could mean any number of things including "invalid email address". Not figuring that out here and treating all HTTP 400 errors as if they were invalid email addresses leads to bad user experience.

Further, the HTTP 429 case is ratelimiting on Input's side. I would treat that as a success and not as a special case of "try again later". Ratelimiting kicks in when the user has already submitted the feedback and is submitting the exact same feedback again.